README.adoc

```diff
@@ -63,16 +63,26 @@ Whether you use built-in repository interfaces or create custom repositories, Ja
 
 ==== Pagination
 
-Jakarta Data also supports parameters to define pagination and sorting.
+Jakarta Data supports two types of Pagination: offset and cursor-based Pagination.
+
+- **Offset Pagination**: Offset Pagination is a popular method for efficiently managing and retrieving large datasets. It is based on dividing the dataset into pages containing a specified number of elements. This method allows developers to retrieve a subset of the dataset by identifying the page number and the maximum number of elements per page.
```

Comment:
This sounds like a definition of "pagination", not of offset-based pagination specifically.

Reply:
@gavinking what do you think?